lighthouse-core/test/gather/computed/metrics/lantern-first-cpu-idle-test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Runner = require('../../../../runner');
+const assert = require('assert');
+
+const trace = require('../../../fixtures/traces/progressive-app-m60.json');
+const devtoolsLog = require('../../../fixtures/traces/progressive-app-m60.devtools.log.json');
+
+/* eslint-env mocha */
+describe('Metrics: Lantern TTFCPUI', () => {
+  it('should compute predicted value', async () => {
+    const artifacts = Runner.instantiateComputedArtifacts();
+    const result = await artifacts.requestLanternFirstCPUIdle({trace, devtoolsLog});
+
+    assert.equal(Math.round(result.timing), 5308);
+    assert.equal(Math.round(result.optimisticEstimate.timeInMs), 2451);
+    assert.equal(Math.round(result.pessimisticEstimate.timeInMs), 2752);
+    assert.equal(result.optimisticEstimate.nodeTiming.size, 19);
+    assert.equal(result.pessimisticEstimate.nodeTiming.size, 79);
+    assert.ok(result.optimisticGraph, 'should have created optimistic graph');
+    assert.ok(result.pessimisticGraph, 'should have created pessimistic graph');
+  });
```

Comment:
it feels like there should be more tests here, but I don't have any special insight as to what boundaries they should be checking. Difficult traces or whatever. They can build up as we come across the difficult cases, but we don't have much infrastructure to spot those :) We should probably get some lantern smoke tests set up.

Reply:
👍 right now its just me manually running my trace-evaluations scripts, post I/O I'll try and stick a lot of my traces onto GCP and get a travis script running to make sure accuracy doesn't regress